addons/microsoft_calendar/utils/event_id_storage.py

```diff
@@ -0,0 +1,10 @@
+IDS_SEPARATOR = ':'
+
+def combine_ids(ms_id, ms_uid):
+    if not ms_id:
+        return False
+    return IDS_SEPARATOR.join([ms_id, ms_uid if ms_uid else ''])
+
+def split_ids(value):
+    ids = value.split(IDS_SEPARATOR)
+    return tuple(ids) if len(ids) > 1 and ids[1] else (ids[0], False)
```

Comment:
Seems unused? Also weird.

Reply:
The provided `value` could be:
- `id1:id2` and in this case a tuple `(id1, id2)` should be returned,
- `id1:` or just `id1` (old format) and in this case a tuple `(id1, False)` should be returned.
That's what this function does but there is probably a better (pythonic) way to write it :-)